MAVProxy/modules/mavproxy_link.py

```diff
@@ -893,6 +882,9 @@ def master_callback(self, m, master):
             for (mod,pm) in self.mpstate.modules:
                 if not hasattr(mod, 'mavlink_packet'):
                     continue
+                # Do not send other-system-or-component heartbeat packets to non-multi-vehicle modules
```

Comment:
Will this break `mavproxy_gimbal.py`?  That's the first module I thought of...

Reply:
Checked - gimbal still works. I'm only _not_ sending heartbeats here